rebar.config.erl

```diff
@@ -159,12 +159,12 @@ is_rocksdb_supported(_) ->
     not is_build_without("ROCKSDB").
 
 project_app_dirs() ->
-    #{edition := Edition, reltype := RelType} = get_edition_from_profile_env(),
-    project_app_dirs(Edition, RelType).
+    Edition = get_edition_from_profile_env(),
```

Comment:
the complexity of this module is necessary to minimize the merge conflict with a private repo which has a flavored packaging based on `reltype`

Reply:
roger that! I reverted the changes back but just left the refactor from `profiles_ce/1`, `profiles_ee/1` and `profiles_dev/1` to `profiles/2`  => `profiles(Edition, RelType)`